shared/actions/fs/index.js

```diff
@@ -530,23 +530,28 @@ function* _loadMimeType(path: Types.Path, refreshTag?: Types.RefreshTag) {
     mimeTypeRefreshTags.set(refreshTag, path)
   }
 
-  const state = yield Saga.select()
-  let localHTTPServerInfo: Types._LocalHTTPServer =
-    state.fs.localHTTPServerInfo || Constants.makeLocalHTTPServer()
+  const state: TypedState = yield Saga.select()
+  let localHTTPServerInfo = state.fs.localHTTPServerInfo || Constants.makeLocalHTTPServer()
```

Comment:
before this was a mix of the actual record from the store, or the non-record value from the service call which makes the typing  way more complicated. instead lets just use the record